src/store/types.ts

```diff
@@ -1,5 +1,22 @@
-import { IAuthStore } from "./auth";
+import { Dispatch } from 'redux';
+
+import { IAuthStore } from './auth';
+import { IUsersStore } from './users';
 
 export type IStore = {
-  auth: IAuthStore
-}
+  auth: IAuthStore;
+  users: IUsersStore;
+};
+
+export type IDataApi = {
```

Comment:
this type should me removed out of here and add should be added to the general types of the project